core/lib/vm_interface/src/lib.rs

```diff
@@ -1,4 +1,21 @@
 //! ZKsync Era VM interfaces.
+//!
+//! # Developer guidelines
```

Comment:
Are these guidelines visible enough? I've thought about placing them in the crate readme, but it would look slightly out of place IMO.

Reply:
I think it's good enough to have them here